build.gradle

```diff
@@ -1,19 +1,24 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
 
-  ext.KOTLIN_VERSION = "1.2.50"
+  ext.KOTLIN_VERSION = "1.2.51"
   ext.ANDROID_PLUGIN_VERSION = "3.2.0-beta02"
 
   repositories {
     google()
     jcenter()
   }
   dependencies {
-    classpath "com.android.tools.build:gradle:3.2.0-beta02"
+    classpath "com.android.tools.build:gradle:3.3.0-alpha04"
```

Comment:
are you using 3.3 canary? I don't know that we should get on the alpha track with this pr, it may be better to stick with 3.2 and merge this once it is released

Reply:
No, I'll revert it back.